runner.js

```diff
@@ -75,5 +75,11 @@ module.exports = function(driver, opts) {
   return GatherScheduler
       .run(gatherers, Object.assign({}, opts, {driver}))
       .then(artifacts => Auditor.audit(artifacts, audits))
-      .then(results => Aggregator.aggregate(aggregators, results));
+      .then(results => Aggregator.aggregate(aggregators, results))
+      .then(aggregations => {
+        return {
+          url: opts.url,
```

Comment:
This is different than how the pretty printed output accesses the URL. They should use the same source (either the existing or from the results of the runner)